web/public/mgmt/config.yml

```diff
@@ -1732,6 +1732,13 @@ collections:
             summary: "{{fundingDefaults}}"
             fields:
               - { label: Issuing Agency Label, name: issuingAgencyText, widget: string }
+          - label: Industry Dropdown
+            name: industryDropdown
+            collapsed: true
+            widget: object
+            summary: "{{industryDropdown}}"
+            fields:
+              - { label: Mapped Industry Copy, name: mappedIndustryCopy, widget: string }
```

Comment:
see my above comment that this should be moved

Reply:
also, I think we could revisit the naming of this field. Remember, the intention of the name and label is to communicate clearly to Jess and Ce what the purpose of the field is and where it gets used. I think we could name this field something that makes it more clear what it means

Reply:
I have a comment below that addresses the naming more broadly btw